### DISCUSSION TITLE
How to get rid of " Trakt Error: No TVDb ID found for ..." messages in the log?

### DISCUSSION THREAD

**Discussion by gitttt54** [meisnate12/Plex-Meta-Manager#2194] [Q&A]
hi all,
I have some trakt lists, which I use for creating nice collections.

unfortunately, the log is flodded with useless informations like:
```
 Trakt Error: No TVDb ID found for 我在古代席卷天下 (2024)                                                  |
| Trakt Error: No TVDb ID found for Incredible Tales (2003)                                          |
| Trakt Error: No TVDb ID found for Bo-Di-La (2022)                                                  |
| Trakt Error: No TVDb ID found for Master Stroke (2020)                                             |
| Trakt Error: No TVDb ID found for Magic (2017)                                                     |
| Trakt Error: No TVDb ID found for The Red House (2016)                                             |
| Trakt Error: No TVDb ID found for 时空迷踪 (2024)                                                      |
| Trakt Error: No TVDb ID found for Kill the wolf flower (2012)                                      |
| Trakt Error: No TVDb ID found for 狂飙1990 (2023)                       
```
I say that they're useless, because I don't have ANY of those movies in my library, so I really do not need an info about those movies (or series), because I can't do anything about it anyways...

my question: Is there a swtich, so I can deactivate those cumbersome messages? they are legion, a couple of hundreds in many lists.

Thank you!

**Comment by chazlarson** [comment]
There is no such switch.

**Reply by gitttt54** [reply]
That really is a pity! So much distracting infos in the log nobody ever can handle or find of any use. The log is so hard to read anyways, now with even more lines to skip while scrolling.

It would be gentle to think about implementation of a little switch to suppress (or write then to another log) whose entries.
